src/scss/src/atoms/Button.scss

```diff
@@ -0,0 +1,133 @@
+@import '../foundation/mixins';
+@import '../foundation/colors';
+@import '../foundation/typography';
+
+.s2s-button {
+  @extend %button-base;
```

Comment:
Why do we need extends and below write the actual classes, why don't implement it here at once?
This styles will be used only here, it's seems unnessesary here, don't you think?

Reply:
You are right. I moved styles from `%button-base` into `s2s-button` directly. But I didn't move other subclasses with variants and sizes because it improves readability and unloads the component to avoid deep nesting.